treeherder/webapp/api/performance_data.py

```diff
@@ -757,8 +757,6 @@ def list(self, request):
         if not query_params.is_valid():
             return Response(data=query_params.errors, status=HTTP_400_BAD_REQUEST)
 
-        startday = query_params.validated_data['startday']
```

Comment:
These fields need to be removed also from `PerfCompareResultsQueryParamsSerializer`